src/main/java/net/starlark/java/annot/StarlarkMethod.java

```diff
@@ -165,6 +165,11 @@
    */
   boolean allowReturnNones() default false;
 
+  /**
+   * When true, Starlark interpreter does not validate function returns a valid Starlark runtime object.
```

Comment:
Please reword for clarity: "If true, for a method which returns {@link Object}, the Starlark interpreter does not validate that the returned object is a legal Starlark value."

Reply:
Done.